src/base/static/components/molecules/map-legend-item.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "react-emotion";
+
+import { Icon } from "../atoms/feedback";
+import { RegularLabel } from "../atoms/typography";
+
+import "./map-legend-item.scss";
+
+const ColorSwatch = styled("span")(props => ({
+  backgroundColor: props.color,
+  width: 20,
+  height: 20,
+  marginRight: 10,
+}));
+
+// TODO: Abstract these components out when we refactor other panel types.
+const IconContainer = styled("div")(props => ({
```

Comment:
nit: Since `props` is not being used here, you can just pass in an object instead of a function.